### DISCUSSION TITLE
Google maps/waze

### DISCUSSION THREAD

**Discussion by paulharman** [tesla-android/issue-tracker#140] [Q&A] [answered]
How can we install google maps or waze on to the pi please?

**Comment by mikegapinski** [comment]
You can install apps with the aurora store, it's a google play client

**Reply by paulharman** [reply]
I tried but waze and google maps do not appear on there?

**Reply by sw1002** [reply]
> You can install apps with the aurora store, it's a google play client

can You give a detailed instruction how to install apps on tesla android and the pi. Is there a documentation for the OS? I am quite unfamiliar with the OS itself, it is quite different, seems like a port from a phone. What I can not do e.g. is typing some word in "search"

**Comment by paulharman** [comment, marked as answer]
got this working now by using apkpure

**Reply by fatbao** [reply]
What GPS are you using?

**Comment by ochompsky** [comment]
wow that's a good idea, i only use this for waze, can cut out the phone part entirely.
